### PR TITLE
Updated button, alert card and back button stories

### PR DESCRIPTION
**Pull request by rachana-egov**

### CODE REVIEW

**Review by naveen-egov — APPROVED** [review]

File: flutter/digit-ui-components/storybook/lib/widgets/atoms/back_button_stories.dart [hunk 1]
```diff
@@ -51,23 +62,12 @@ List<Story> digitBackButtonStories() {
...
-                Icons.arrow_left,
-                size: MediaQuery.of(context).size.width < 500
-                    ? Theme.of(context).spacerTheme.spacer5
-                    : Theme.of(context).spacerTheme.spacer6,
-                color: Theme.of(context).colorTheme.primary.primary2,
-              ),
-            ),
             label: context.knobs.text(label: 'Label', initial: 'Back'),
           ),
         );
```
[the remark this answers is not in the document]
Comment (rachana-egov) [reply flutter/digit-ui-components/storybook/lib/widgets/atoms/back_button_stories.dart:73]: that is there to have a different configuration

File: flutter/digit-ui-components/storybook/lib/iframe/iframe_widget.dart [hunk 2]
```diff
@@ -33,38 +33,52 @@ class IframeWidget extends StatelessWidget {
             '''
               (function() {
                 try {
-                  // Find the tab list element
+                  // Find the main content element
+                  var mainContent = document.querySelector('.flex-1.relative.py-8.lg\\:px-12.break-anywhere.page-api-block\\:xl\\:max-2xl\\:pr-0.page-api-block\\:max-w-\\[1654px\\].page-api-block\\:mx-auto');
+                  
+                  if (mainContent) {
+                    // Remove all other elements from the body except the main content
+                    var allChildren = Array.from(document.body.children);
```
[the remark this answers is not in the document]
Comment (rachana-egov) [reply flutter/digit-ui-components/storybook/lib/iframe/iframe_widget.dart:41]: we do not have access to change class name that's why i have added try catch bloc

**Comment by naveen-egov** [comment]
@coderabbitai can you approve the pr

**Review by naveen-egov — APPROVED** [review]

File: flutter/digit-ui-components/storybook/lib/widgets/atoms/Info_card_stories.dart [hunk 3]
```diff
@@ -1,128 +1,354 @@
...
+                ...List.generate(
+                  5,
+                      (_) => Image.network(
+                    'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQIGMLufj86aep95KwMzr3U0QShg7oxdAG8gBPJ9ALIFQ&s',
+                  ),
+                ),
+              if (showInfoButton)
+                Column(
+                  children: [
+                    if (showInfoButton && showInlineWidget == true)
```
[the remark this answers is not in the document]
Comment (naveen-egov) [reply flutter/digit-ui-components/storybook/lib/widgets/atoms/Info_card_stories.dart:70]: by default its false this case don't occur

**Pull request merged**